mike/commands.py

```diff
@@ -18,7 +18,7 @@ def _redirect_template(user_template=None):
     f = (open(user_template, 'rb') if user_template else
          resource_stream(__name__, 'templates/redirect.html'))
     with f:
-        return Template(f.read().decode('utf-8'), autoescape=True)
+        return Template(f.read().decode('utf-8'), autoescape=True, keep_trailing_newline=True)
```

Comment:
It looks like `keep_trailing_newlines` was added in Jinja 2.7. Could you update the `setup.py` requirement to look for `jinja2 >= 2.7`? (Technically, `mkdocs` requires that already, but I think it'd be good to be explicit here.)

This also fails to pass lint, since the line is too long:
```suggestion
        return Template(f.read().decode('utf-8'), autoescape=True,
                        keep_trailing_newline=True)
```

Reply:
Both fixes will be in updated commit.